src/app/infokit/infokit.pipe.ts

```diff
@@ -0,0 +1,48 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe ({
+  name: 'infokitPipe'
+})
+
+export class InfokitPipe implements PipeTransform {
+  transform(dict: any[], args: any= []) {
+    let activity: any[] = [];
+    Object.keys(dict).forEach(key => {
+        switch (key) {
+          case 'malaria_def' : {
+            activity.push({key: key, def: 'Malaria', val: dict[key]});
```

Comment:
if the dict has def & val, then you don't even these all these switch statements, and can be simplified to:

```
activity.push({key: key, def: dict[key].def, val: dict[key].val});
```